.github/workflows/ansible-release.yml

```
@@ -84,24 +111,26 @@ jobs:
 
       - name: Commit ansible-build-data and push the changes to github
         working-directory: >-
-          antsibull/build/ansible-build-data/${{ inputs.ansible-major-version }}
+          antsibull/build/ansible-build-data/${{ steps.extract-version.outputs.major-version }}
         run: |
          git add .
          git commit -m "${CI_COMMIT_MESSAGE}"
          git push origin "publish-${ANSIBLE_VERSION}"
 
       - name: Create PR to the ansible-build-data
+        id: create-pr
         working-directory: antsibull/build/ansible-build-data
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           ARTIFACT_URL: ${{ steps.upload-artifact.outputs.artifact-url }}
         run: |
           body="$(echo -e "${CI_COMMIT_MESSAGE}\nRelease artifacts: <${ARTIFACT_URL}>")"
+          echo -n "pr_url=" >> "$GITHUB_OUTPUT"
           gh pr create \
           --base main \
           --head "publish-${ANSIBLE_VERSION}" \
           --title "Release Ansible ${ANSIBLE_VERSION}" \
-          --body "${body}"
+          --body "${body}" >> "$GITHUB_OUTPUT"
```

I would use tee so the PR url is still included in the output like before

 ```suggestion
          --body "${body}" | tee -a "$GITHUB_OUTPUT"
```